pipeline/sideload.go

```diff
@@ -35,6 +35,9 @@ type SideloadNode struct {
 	// Tags is a list of tags to load.
 	// tick:ignore
 	Tags map[string]string `tick:"Tag" json:"tags"`
+
+	HttpUser     string `json:"httpuser"`
+	HttpPassword string `json:"httppassword"`
```

Comment:
I'd prefer these not be part of the TICKscript so that we do not encourage users to place credentials in their scripts.

There is already a concept of `httppost` endpoints that allow the configuration of URLs and credentials, should we leverage that?

Reply:
Yes, I had not considered the use of endpoints. That would be abetter way to handle authentication.

I would assume that the ordering would work since I tried to implement it as an absrraction of the existing code, but I will definitely test it.

Reply:
Would it be better to create a new configuration section for the sideload, try and re-use what is available for httppost, or create a more generic endpoint configuration?

The httppost configuration could be used unmodified, I think, but then the dual uses of httppost in the configuration would be obfuscated and confusing.